netbox/netbox/views/generic/object_views.py

```diff
@@ -234,6 +234,8 @@ def get(self, request, *args, **kwargs):
         # If this is an HTMX request, return only the rendered form HTML
         if htmx_partial(request):
             return render(request, self.htmx_template_name, {
+                'model': model,
```

Comment:
What are these used for?

Reply:
Just additional context for the template, should they be needed.